compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAssignTiles.cpp

```diff
@@ -373,7 +396,7 @@ class AMDAIEAssignTilesPass
     : public impl::AMDAIEAssignTilesBase<AMDAIEAssignTilesPass> {
  public:
   void getDependentDialects(DialectRegistry &registry) const override {
-    registry.insert<AMDAIEDialect>();
+    registry.insert<AMDAIEDialect, arith::ArithDialect>();
```

Comment:
Some test must've triggered this addition as it wasn't caught earlier? If yes, then it'd be nice to add a short lit test for this as well.

Reply:
No, this isn't triggered and I thought this was needed, but I can remove this as well and it works, which is unclear to me.